### PR TITLE
Update to clap 3.1.x

### PR DESCRIPTION
**Pull request by Sciencentistguy**
Updated to use clap 3.1.x instead of structopt.

### CODE REVIEW

**Review by conradludgate — DISMISSED** [review]
You beat me to it 😅 Looks good, thanks!

File: src/command/history.rs [hunk 1]
```diff
@@ -2,60 +2,51 @@ use std::env;
...
+        #[clap(long, short)]
         cwd: bool,
 
-        #[structopt(long, short)]
+        #[clap(long, short)]
         session: bool,
 
-        #[structopt(long, short)]
+        #[clap(long, short)]
         human: bool,
```
Comment (conradludgate) [inline src/command/history.rs:36]: Seems this is having some troubles. It's inferring the subcommand as `-h` but it's conflicting with the help option. There might be other cases like this too.

I guess an easy solution would be to extract this list variant into a new struct

Comment (Sciencentistguy) [reply src/command/history.rs:36]: I think we have to make a decision here; do we want `atuin history last -h` to be `atuin history last --help` or `atuin history last --human`?

Comment (ellie) [reply src/command/history.rs:36]: I'd go for -h being help, it's more intuitive and in line with what people might expect

Comment (Sciencentistguy) [reply src/command/history.rs]: Agreed

**Comment by conradludgate** [comment]
Seems the same issue applies with the `atuin search` subcommand (again a `--human` flag). I also agree that it should be `--long` only

**Review by conradludgate — APPROVED** [review]
Thanks! 🥳

**Pull request merged**